<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SelectorThing</class>
 <widget class="QWidget" name="SelectorThing">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>376</width>
    <height>287</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="lblSelectedThing">
       <property name="text">
        <string>Tryk på knappen &quot;Nyt udfald&quot;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="cbShowChoosingAnimation">
     <property name="text">
      <string>Vis &quot;vælger&quot; animation</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0">
     <item>
      <widget class="QPushButton" name="btnNewOutcome">
       <property name="text">
        <string>Nyt udfald</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnManageOptions">
       <property name="text">
        <string>Håndter muligheder</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
